components/closebutton/metadata/mods.md

```diff
@@ -1,33 +1,28 @@
-| Modifiable custom properties                        |
-| --------------------------------------------------- |
-| `--mod-animation-duration-100`                      |
-| `--mod-button-animation-duration`                   |
-| `--mod-button-font-family`                          |
-| `--mod-button-line-height`                          |
-| `--mod-closebutton-align-self`                      |
-| `--mod-closebutton-animation-duraction`             |
-| `--mod-closebutton-animation-duration`              |
-| `--mod-closebutton-background-color-default`        |
-| `--mod-closebutton-background-color-down`           |
-| `--mod-closebutton-background-color-focus`          |
-| `--mod-closebutton-background-color-hover`          |
-| `--mod-closebutton-border-radius`                   |
-| `--mod-closebutton-focus-indicator-color`           |
-| `--mod-closebutton-focus-indicator-gap`             |
-| `--mod-closebutton-focus-indicator-thickness`       |
-| `--mod-closebutton-height`                          |
-| `--mod-closebutton-icon-color-default`              |
-| `--mod-closebutton-icon-color-disabled`             |
-| `--mod-closebutton-icon-color-down`                 |
-| `--mod-closebutton-icon-color-focus`                |
-| `--mod-closebutton-icon-color-hover`                |
-| `--mod-closebutton-margin-inline`                   |
-| `--mod-closebutton-margin-top`                      |
-| `--mod-closebutton-size`                            |
-| `--mod-closebutton-static-background-color-default` |
-| `--mod-closebutton-static-background-color-down`    |
-| `--mod-closebutton-static-background-color-focus`   |
-| `--mod-closebutton-static-background-color-hover`   |
```

Comment:
Some of these are going away. I figured it would be ok to just remove instead of following a deprecation flow for these since this is a breaking change already, but I'm open to thoughts!

Reply:
I think these are fine to be going away; I saw the bg color ones would just use the existing mods and don't need a static specific mod. And the size was not changing anything. It'd probably be a good idea to document these removals in the PR at least, especially the bg colors and the alternative mod name that takes their place.

Reply:
Good call, I've added those to the PR description 👍